docs/components/movement-sensor/_index.md

```diff
@@ -0,0 +1,672 @@
+---
+title: "Movement Sensor Component"
+linkTitle: "Movement Sensor"
+weight: 70
+type: "docs"
+description: "A sensor that measures location, kinematic data, or both."
+tags: ["movement sensor", "gps", "imu", "sensor", "components"]
+icon: "/components/img/components/imu.svg"
+no_list: true
+aliases:
+    - /components/movement-sensor/
+# SME: Rand
+---
+
+A movement sensor component is a sensor that gives data on where a robot is and how fast it is moving.

```

Comment:
```suggestion
A movement sensor is a sensor that gives data on where a robot is and how fast it is moving.
```

Reply:
I included the word "component" to make it more clear that a "movement sensor" is a Viam concept, not a common term.